packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -129,22 +129,44 @@
   src: url("/fonts/okticon.eot?#iefix") format("embedded-opentype"), url("/fonts/okticon.woff") format("woff"), url("/fonts/okticon.ttf") format("truetype"), url("/fonts/okticon.svg#okticon") format("svg");
 }
 
+html {
+  font-size: 16px;
```

Comment:
nit: should we use `$base-font-size` here?

Reply:
This is already changed to `$base-font-size`. 🤔

Reply:
ah I was looking at the first commit, sry!